pkgs/development/tools/language-servers/csharp-ls/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, buildDotnetGlobalTool
+, dotnetCorePackages
+}:
+let
+  inherit (dotnetCorePackages) sdk_7_0;
```

Comment:
This let in is not that useful. Either we write sdk_7_0 to a variable that stays static or we can inline it.

Reply:
Should I create a PR for this ? (I don't mind doing so)

Reply:
You don't need to, just something I noticed which can be changed in the future.